docs/transports.md

```diff
@@ -467,6 +468,23 @@ PRs to this document are welcome for any new transports!
 + [pino-syslog](#pino-syslog)
 + [pino-websocket](#pino-websocket)
 
+<a id="@appsignal/nodejs-pino"></a>
+### @appsignal/nodejs
+
+[@appsignal/nodejs](https://www.npmjs.com/package/@appsignal/nodejs) [`AppsignalPinoTransport`](https://github.com/appsignal/appsignal-nodejs/blob/337eed9b72194f9500a685410fbfc9197c4db911/src/pino_transport.ts) is the official [AppSignal](https://appsignal.com/) transport for Pino.
```

Comment:
@luismiramirez I see that you have had to update this PR at least once merely to update the hash in the URL. Can you provide a URL that doesn't need to be updated so often? It would be better if the module were in a repo dedicated to its maintenance.